scripts/devenv-builder/configure-composer.sh

```diff
@@ -95,6 +95,25 @@ enable_beta_or_eus_repositories() {
     # The configuration will remain unchanged for non-beta and non-EUS operating systems.
     if grep -qE "Red Hat Enterprise Linux.*Beta" /etc/redhat-release; then
         sudo sed -i "s,dist/rhel${version_id_major}/${version_id},beta/rhel${version_id_major}/${version_id_major},g" "${composer_config}"
+    elif dnf repolist | grep -q download.eng.brq.redhat.com; then
```

Comment:
I'm not sure this should be an `elif`. Should we better put it as its own clause?

Reply:
I made it an elif because I think we would only want the composer system source to use either a Beta url or this internal download.eng.brq.redhat.com url but not both. It's also fine to put it in it's own clause, since composer_config is being over written for this special case.